Tests/Tests/Saml2StatementExtensionsTests.cs

```diff
@@ -31,6 +31,17 @@ public void Saml2StatementExtensions_ToXElement_TypeCheck()
                 .And.Message.Should().Be("Statement of type StubSaml2Statement is not supported.");
         }
 
+        [TestMethod]
```

Comment:
What does this test that isn't already covered by other tests?

Reply:
According to coveralls.io, code is not fully covered without this test:

https://coveralls.io/builds/13864802/source?filename=%2FKentor.AuthServices%2FSaml2StatementExtension.cs